server/features/EmailInbox/EmailInbox.ts

```diff
@@ -54,8 +56,9 @@ export async function configureEmailInboxes(): Promise<void> {
 			try {
 				await EmailMessageHistory.insertOne({ _id: email.messageId, email: emailInboxRecord.email });
 				onEmailReceived(email, emailInboxRecord.email, emailInboxRecord.department);
-			} catch (e) {
+			} catch (e: any) {
 				// In case the email message history has been received by other instance..
+				logger.error(e.message);
```

Comment:
if you have an error object, you can log it entirely
```
logger.error(e);
```

if for some reason TS complains about it, you can add the error object to the `err` key
```
logger.error({ err: e });
```